### PR TITLE
fix(buttons): fixed loader on friends cards

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Substitue buttons loader in the friends cards with the loader that is already in the chat and main screen
**Which issue(s) this PR fixes** 🔨
AP-2058

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/interactables/Button/Button.vue [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default Vue.extend({
      */
     loadingText: {
       type: String,
-      default: 'Loading...',
+      default: null,
```
Comment (molimauro) [inline components/interactables/Button/Button.vue:76]: is this text still needed?

Comment (ThomBos) [reply components/interactables/Button/Button.vue:76]: mmhh need to check around the app if this loader is triggered.... going for it

File: assets/styles/base.less [hunk 2]
```diff
@@ -640,6 +640,15 @@ button {
...
 
+.friend {
+  .loader-container {
+    width: 60px;
+    .loading-bar .bar {
+      animation: shorterloading 2s cubic-bezier(0.17, 0.37, 0.43, 0.67) infinite;
+    }
+  }
+}
+
```
Comment (josephmcg) [inline assets/styles/base.less:651]: we want to slim down this file as much as possible, could you add this to scoped component css?

Comment (ThomBos) [reply assets/styles/base.less:651]: i've tried to do that, but it couldn't overwrite the animation

Comment (josephmcg) [reply assets/styles/base.less:651]: Sounds like its better off as a new loading component then. Or if its really similar to the other one, you can pass a prop

Comment (josephmcg) [reply assets/styles/base.less:651]: If its using the extend stuff, that often complicates the css priority. I try to avoid if possible

Comment (molimauro) [reply assets/styles/base.less:651]: I'd avoid as well, just use a prop

Comment (ThomBos) [reply assets/styles/base.less:651]: yeah the prop is definitely better, thanks for clarifying! 👍

File: components/ui/Loaders/LoadingBar/LoadingBar.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="loading-bar">
+  <div class="loading-bar" :class="short ? 'short' : ''">
```
Comment (josephmcg) [inline components/ui/Loaders/LoadingBar/LoadingBar.vue:2]: a ternary is good if there are two values that could be applied. Try this syntax instead
```
:class="{short : short}"
```

Comment (ThomBos) [reply components/ui/Loaders/LoadingBar/LoadingBar.vue]: it also looks much better while reading the code 👍

**Pull request merged**